datafusion/sqllogictest/src/test_context.rs

```diff
@@ -91,13 +91,18 @@ impl TestContext {
                 {
                     info!("Registering avro tables");
                     register_avro_tables(&mut test_ctx).await;
+                    test_ctx.ctx = test_ctx.ctx.enable_url_table();
                 }
                 #[cfg(not(feature = "avro"))]
                 {
                     info!("Skipping {file_name} because avro feature is not enabled");
                     return None;
                 }
             }
+            "describe.slt" | "arrow_files.slt" | "csv_files.slt" | "json.slt"
```

Comment:
I think it would help to point out some rationale

```suggestion
           // For (only) these files, enable the dynamic file catalog
            "describe.slt" | "arrow_files.slt" | "csv_files.slt" | "json.slt"
```

Reply:
All the tests have been moved to `dynamic_file.slt`.